extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -149,16 +149,40 @@
     @ConfigItem(defaultValue = "true")
     public boolean secondLevelCachingEnabled;
 
+    /**
+     * Defines the method for multi-tenancy (DATABASE, NONE, SCHEMA). The complete list of allowed values is available in the
+     * https://docs.jboss.org/hibernate/stable/orm/javadocs/org/hibernate/MultiTenancyStrategy.html[Hibernate ORM JavaDoc].
+     * The type DISCRIMINATOR is currently not supported. The default value is NONE (no multi-tenancy).
+     *
+     * @asciidoclet
+     */
+    @ConfigItem
+    public Optional<String> multitenant;
+
+    /**
+     * Defines the name of the datasource to use in case of SCHEMA approach. The datasource of the persistence unit will be used
+     * if not set.
+     */
+    @ConfigItem
+    public Optional<String> multitenantSchemaDatasource;
```

Comment:
@michael-schnell @Sanne I'm not entirely sure this is useful now that we can define a datasource for a PU. Or in the case of the schema do you want to use another datasource when the tenant is defined?

Reply:
AFAICT there is actually zero difference between `SCHEMA` and `DATABASE` in Hibernate multitenancy, so I don't think you need two things.

Reply:
I don't know about Hibernate ORM but it makes a difference here as things are handled in the Quarkus extension. I'm unclear though if we really want this.

I agree this needs more discussions and thoughts.

Reply:
There might not be a difference within Hibernatate between SCHEMA and DATABASE, but there is one if you need to generate to database schema with Flyway.